src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -335,6 +435,41 @@ class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with Flat
     // check that the instances are still in the DB (they get removed by the ClusterMonitorActor)
     val instances = dbFutureValue { _.instanceQuery.getAllForCluster(getClusterId(clusterCreateResponse)) }
     instances.toSet shouldBe Set(masterInstance, workerInstance1, workerInstance2)
+
+    // ------------------ Grossly repeating above using v2 of the cluster creation API :( ------------------
```

Comment:
To be able to use `forallClusterCreationMethods` here, the method would have to be bloated to account for distinct `master/workerInstance` between `v1` and v2` API calls, which didn't seem worth it.